### PR TITLE
Pull pacing_type from Studio

### PR DESCRIPTION
**Pull request by mikix**
When an org is on the new publisher, pull it in from Studio
and stop pushing it on updates. (i.e. move source-of-edit to
Studio for users of frontend-app-publisher)

https://openedx.atlassian.net/browse/DISCO-1341

This is related to https://github.com/edx/frontend-app-publisher/pull/258

### CODE REVIEW

**Comment by mikix** [comment]
Note: this shouldn't land until the frontend work is done as well.

File: course_discovery/apps/api/utils.py [hunk 1]
```diff
@@ -177,9 +178,11 @@ def generate_data_for_studio_api(cls, course_run, creating, user=None):
             'number': number,
             'run': run,
             'team': team,
-            'pacing_type': cls._run_pacing(course_run),
         }
 
+        if pacing:
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/api/utils.py:183]: I don't love all of these ifs, but I don't have a better solution.

File: course_discovery/apps/publisher/utils.py [hunk 2]
```diff
@@ -180,6 +180,26 @@ def is_on_new_pub_fe(user):
...
 
+def is_course_on_new_pub_fe(course):
+    """
+    Returns True if all the course's organizations have been moved to new publisher frontend
+
+    Args:
+        course: A course_metadata Course object (not a Publisher one)
+    """
+    try:
+        orgs_on_new_pub_fe = frozenset(settings.ORGS_ON_NEW_PUB_FE.split(','))
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/publisher/utils.py]: Is there not a cleaner way to do this without a try except? Like any sort of `getattr(settings, 'ORGS_ON_NEW_PUB_FE')` or something? Feels heavy handed to be waiting for an AttributeError when checking a setting.

**Review by Dillon-Dumesnil — APPROVED** [review]
Sure

File: course_discovery/apps/publisher/utils.py (hunk 2 above)
Comment (mikix) [reply course_discovery/apps/publisher/utils.py]: Fair. I stole this format from `is_on_new_pub_fe`, because I was lazy. Getattr is better. I'll do that.

Comment (mikix) [reply course_discovery/apps/publisher/utils.py]: Done. And fixed the other function to follow suit.

**Pull request merged**